pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -391,6 +397,37 @@ func (nc *Controller) Run(stopCh <-chan struct{}) {
 		go nc.taintManager.Run(wait.NeverStop)
 	}
 
+	if nc.taintNodeByCondition {
+		// TODO(k82cn): find a reasonable worker size and channel length.
```

Comment:
this is consistent with 8 in taint manager. Since that number seems to work, maybe simply remove that todo and use 8 here too.
Ideally, this may be some constant for the whole controller and shared between those two.

Reply:
whats special about 8 ?